orangecontrib/text/widgets/owconcordance.py

```diff
@@ -127,6 +128,7 @@ def _compute_word_index(self):
             self.word_index = [
                 (doc_idx, offset) for doc_idx, doc in enumerate(self.indices)
                 for offset in doc.offsets(self.word)]
+            self.colored_rows = set(sorted({d[0] for d in self.word_index})[::2])
```

Comment:
For the sake of purity, could you add `self.colored_rows = None` (or have `self.colored_rows = self.word_index = None`) in the above case? It won't change anything since there are 0 rows anyway, but it's cleaner.